### PR TITLE
fetching session data with Kobiton REST API

### PR DESCRIPTION
**Pull request by phanthaoIT**

### CODE REVIEW

File: rest-api/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    >Please visit here to install https://nodejs.org/en/download/.
+
+  - Kobiton Username and API key
+    >Please visit our [blog](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/) and navigate to `IV. Configure Test Scripts for Kobiton` for more details in getting Username and API key.
+
+## Kobiton REST API command
+  This repository includes samples of simple NodeJS app for fetching test session data with Kobiton REST API.
+
+  The sample contains scripts:
+  - [get-session-data.js](https://github.com/kobiton/samples/rest-api/get-session-data.js): Fetch a session data
```
Comment (tiennguyenhoan) [inline rest-api/README.md]: what happen if we use
```
[get-session-data.js](./get-session.data.js) 
```

File: rest-api/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,48 @@
+parser: babel-eslint
```
Comment (tiennguyenhoan) [inline rest-api/.eslintrc]: don't push this file

File: rest-api/README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  - [get-session-commands.js](https://github.com/kobiton/samples/rest-api/get-session-commands.js): Fetch all commands of a session
```
Comment (tiennguyenhoan) [inline rest-api/README.md]: Same

File: rest-api/README.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+- Request command: 
+
+    `GET /sessions/{sessionId}`
+
+ Retrieve a session info belonging to the current user or organization.
+> Note: Absolute urls in response body like *"screenshotDownloadUrl"*, *"previewUrl"*, etc. expire in 2 hours since the response is made.
+
+
+- Execute:
```
Comment (tiennguyenhoan) [inline rest-api/README.md]: - **Execute: **

File: rest-api/README.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+
+- Execute: 
+    ```bash
+    $ USERNAME=<YOUR_KOBITON_USERNAME> \
+    > API_KEY=<YOUR_KOBITON_API_KEY> \
+    > SESSION_ID=<SESSION_ID> \
+    > node get-session-data.js 
+    ```
+  **Example:**
```
Comment (tiennguyenhoan) [inline rest-api/README.md]: I just think that we don't need to bold this, what do you think ? 

```
**bold**
notbold
```
- Result: 
**bold**
notbold

File: rest-api/README.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+  ``` bash
+   $ USERNAME=kobitonTester \
+    > API_KEY=90bc23f4-58s3-4ds3-223a-ds42d45c085cf \
+    > SESSION_ID=3894 \
+    > node get-session-data.js 
+  ```
+  **Example result:**
```
Comment (tiennguyenhoan) [inline rest-api/README.md]: Same

File: rest-api/README.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+      "isCloud" :  true,
+      "executionData" : { ... },
+      "log" : { ... },
+      "video" : { ... }
+      }
+    ```
+
+For more information, please visit https://api.kobiton.com/docs/#get-a-session.
+
+## Get session commands
```
Comment (tiennguyenhoan) [inline rest-api/README.md:77]: Same above

**Review by tiennguyenhoan — CHANGES_REQUESTED** [review]
@phanthaoIT I have some question for the readme

**Comment by tiennguyenhoan** [comment]
@phanthaoIT still have one more things, the **first lette**r of **each commit** should be in UPPERCASE. Group all commit into one before ping me to review

File: rest-api/README.md (hunk 5 above)
Comment (phanthaoIT) [reply rest-api/README.md]: yeah, it looks better

**Pull request closed without merging**